### DISCUSSION TITLE
Domain login credentials

### DISCUSSION THREAD

**Discussion by cherrywebdesignstudio** [vercel/community#6699] [Help] [answered]
### Summary

Hello community. In my web-app, there's a set of environmental variable called ADMIN_ID & ADMIN_PASSWORD on asking my developer, he sys these are the credentials for logging in into domain/route. What exactly this could be? And where do I get that to change the settings? On asking varcel support they lead me to DNS management page where I see no such credentials. Looking for some help here as an owner with no knowledge of development.

### Steps to Reproduce

This is a general question regarding domain/routing credentials.

**Comment by versecafe** [comment, marked as answer]
@cherrywebdesignstudio If they are environment variables you can change them in either:
`https://vercel.com/[team]/[project]/settings/environment-variables` project env vars
`https://vercel.com/[team]/~/settings/environment-variables` team env vars

**Reply by amyegan** [reply]
@cherrywebdesignstudio I think @versecafe has the right idea. These are most likely environment variables specific to your web app. You can learn more about how to manage them in the [Environment Variables](https://vercel.com/docs/projects/environment-variables#) documentation and in the guide called [How do I add environment variables to my Vercel project?](https://vercel.com/guides/how-to-add-vercel-environment-variables)
